Tests/Functional/FailingTestSuite/main.swift

```diff
@@ -10,9 +10,7 @@
 
 class PassingTestCase: XCTestCase {
     static var allTests: [(String, PassingTestCase -> () throws -> Void)] {
-        return [
-            ("test_passes", test_passes),
-        ]
+        return [("test_passes", test_passes)]
```

Comment:
I can agree on the 'one element per line', but I disagree on the trailing comma. Before this comment, I was not aware that a trailing comma was legal syntax. It could therefore be confusing to newcomers (or even people who were unaware of this).